package.json

```diff
@@ -504,7 +520,11 @@
       },
       {
         "view": "SonarLint.ConnectedMode",
-        "contents": "No SonarQube/SonarCloud connections found [learn more](https://marketplace.visualstudio.com/items?itemName=SonarSource.sonarlint-vscode#connected-mode).\n[Add SonarQube Connection](command:SonarLint.ConnectToSonarQube)\n[Add SonarCloud Connection](command:SonarLint.ConnectToSonarCloud)"
+        "contents": "No SonarQube/SonarCloud connections found. [Learn more](https://marketplace.visualstudio.com/items?itemName=SonarSource.sonarlint-vscode#connected-mode).\n[Add SonarQube Connection](command:SonarLint.ConnectToSonarQube)\n[Add SonarCloud Connection](command:SonarLint.ConnectToSonarCloud)"
+      },
+      {
+        "view": "SonarLint.SecurityHotspots",
+        "contents": "[Security Hotspots](https://docs.sonarqube.org/latest/user-guide/security-hotspots/) detected by SonarLint and SonarQube will be visible here if you have set up SonarLint Connected Mode with SonarQube v9.7 or higher. [Learn more](https://marketplace.visualstudio.com/items?itemName=SonarSource.sonarlint-vscode#connected-mode)."
```

Comment:
Possibly we need to use a link to the new doc content here, specifying how Hotspots work in SonarLint